include/snitch/snitch_registry.hpp

```diff
@@ -106,12 +108,17 @@ struct registered_reporter {
 };
 
 template<typename T>
-concept reporter_type =
-    requires(registry& reg) { T{reg}; } &&
-    requires(T& rep, registry& reg, std::string_view k, std::string_view v) {
-        { rep.configure(reg, k, v) } -> convertible_to<bool>;
-    } && requires(T& rep, const registry& reg, const event::data& e) { rep.report(reg, e); };
+concept reporter_type = requires(registry& reg) {
+    T{reg};
+}
+&&requires(T& rep, registry& reg, std::string_view k, std::string_view v) {
+    { rep.configure(reg, k, v) } -> convertible_to<bool>;
+}
+&&requires(T& rep, const registry& reg, const event::data& e) {
+    rep.report(reg, e);
+};
```

Comment:
You are probably using a slightly older `clang-format` which doesn't understand `concept` properly. The formatting is off here and should be reverted please.

Reply:
It should be ok now, please check